istioctl/cmd/root.go

```diff
@@ -251,6 +251,10 @@ debug and diagnose their Istio mesh.
 	hideInheritedFlags(manifestCmd, "namespace", "istioNamespace", "charts")
 	rootCmd.AddCommand(manifestCmd)
 
+	meshInstallCmd := mesh.InstallCmd(loggingOptions)
+	deprecate(meshInstallCmd)
```

Comment:
This isn't needed anymore. As it will show deprecation msg for `istioctl install`. Please remove it.

Reply:
Hmm... Sorry I got confused.

It shows deprecation msg for `istioctl manifest install` not `istioctl install` because it adds to `manifestCmd` not `rootCmd` by [`manifestCmd.AddCommand(meshInstallCmd)`](https://github.com/istio/istio/pull/27277/files#diff-23cb7ef0d35434bca82aaa95a392ce54R256).

But as per your comment, we don't need deprecated message for `istioctl manifest install` at all? if so, we should not move [meshInstallCmd](https://github.com/istio/istio/pull/27277/files#diff-23cb7ef0d35434bca82aaa95a392ce54R254-R256) in `root.go`?

Reply:
Just in case, install command is already registered in root command as https://github.com/istio/istio/blob/master/istioctl/cmd/root.go#L258-L260.

Reply:
Ahh, well in that case we don't need to register it again.

Reply:
> if so, we should not move meshInstallCmd in root.go but just remove it from istioctl mesh?

Right.

Reply:
I got it. Thank you! Updated.